chsdi/static/vectorStyles/ch.bfe.ladestellen-elektromobilitaet.json

```diff
@@ -1,68 +1,261 @@
 {
 	"type": "unique",
-	"property": "Available",
+	"property": "Availability",
 	"values": [{
+                "geomType": "point",
+                "value": "Unknown",
+                "minResolution":0,
+                "maxResolution":20,
+                "vectorOptions": {
+                        "type": "icon",
+                "src": "https://data.int.bgdi.ch/ch.bfe.ladestellen-elektromobilitaet/symbols/icon-pin-shadow-no-status-information-50px.png",
```

Comment:
A question for @ltkum 
Is there a reason to have the icons on data.int and not data.geo?
If not, just tell and I will take care of this small fix